app/views/forecast_snapshots/show.html.erb

```diff
@@ -14,11 +14,12 @@
 
   // Create array of series objects
   var all_data = [];
-
   <% @forecast_snapshot.history_tsd.get_all_series.each do |series| %>
+  <% @mnemonic = series[:name] + '.' + series[:frequency] %>
```

Comment:
Technically, `@mnemonic` should be `mnemonic`. With the `@`, it becomes an instance variable of the current forecast_snapshot and lives in memory as long as the forecast_snapshot.